build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenLocal()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```

Comment:
Any reason why this can't be bumped to higher versions like `3.0.0`? As per its [release notes](https://developer.android.com/studio/releases/gradle-plugin.html#3-0-0), `3.0.0` offers better compilations speeds and requires Build Tools 26.0.2 or higher.

Reply:
Also, shouldn't the `gradle-wrapper.properties` `distributionUrl` be bumped?

Reply:
Bumping to the 3.0.0 is out of the scope for this PR. IMO, should be done in another PR.